mozilla/pdf.js
# Detect scripted auto-print requests

Fixes #6106

As usual, see the commit mesage for a detailed explanation.

To test whether the changes work:
1. node server
2. Visit http://localhost:8888/test/unit/unit_test.html?spec=api

To manually test whether the changes work:
1. node server
2. Open http://localhost:8888/web/viewer.html?file=/test/pdfs/bug1001080.pdf  
   and check whether a print dialog opens
3. Idem for http://localhost:8888/web/viewer.html?file=/test/pdfs/issue6106.pdf
